snapm/manager/plugins/lvm2.py

```
@@ -1402,17 +1427,14 @@ def discover_snapshots(self):
                             timestamp,
                             mount_point,
                             self,
-                            f"{lv_dict[LVS_VG_NAME]}",
+                            f"{vg_name}",
                             f"{lv_name}",
                             lv_dict=lv_dict,
                         )
                     )
 
         for snapshot in snapshots:
```

Ruff (0.15.9)

[warning] Loop control variable `snapshot` not used within loop body

Rename unused `snapshot` to `_snapshot`

(B007)